content/intro-to-storybook/vue/fr/conclusion.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
+
+[📕 **Projet GitHub : chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/vue)
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
+Storybook est un puissant outil pour React, React Native, Vue, Angular, Svelte et beaucoup d'autres frameworks. Il y a une communauté de développeurs prospère et une nombre important d'addons. Cette introduction égratigne la surface de ce qui est possible. Nous sommes convaincus qu'une fois que vous aurez adopté Storybook, vous serez impressionnés par la manière de produire des interfaces utilisateurs durables.
+
+## En apprendre plus
+
+Vous voulez en apprendre plus ? Voici des ressources utiles :
+
+- [**Documentation Officielle de Storybook**](https://storybook.js.org/basics/introduction/) possède une documentation de l'API, des liens de la communauté et une galerie d'addons.
+
+- [**The Delightful Storybook Workflow**](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07) met en évidence les meilleures pratiques de workflow utilisées par les équipes de haute vélocité de Squarespace, Major League Soccer, Discovery Network, et Apollo GraphQL.
+
+- [**Visual Testing Handbook**](https://www.chromatic.com/book/visual-testing-handbook) plonge profondément dans l'utilisation de Storybook pour visualiser les composants de test. Livre numérique gratui de 31 pages.
```

Comment:
@jeanphibaconnais when you're able can you please adjust the link for "Visual Testing Handbook" to point at https://www.learnstorybook.com/visual-testing-handbook/ ?

Reply:
it's ok for this one.

Reply:
Once again thank you for making the change